addons/stock/views/stock_package_views.xml

```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="package_search_view" model="ir.ui.view">
+        <field name="name">stock.package.search</field>
+        <field name="model">stock.package</field>
+        <field eval="10" name="priority"/>
+        <field name="arch" type="xml">
+            <search string="Package">
+                <field name="name" string="Package Name"/>
+                <field name="location_id"/>
+                <field name="package_type_id"/>
+                <filter string="In internal locations" name="internal" domain="['|', ('location_id.usage', '=', 'internal'), ('location_id', '=', False)]" groups="stock.group_stock_multi_locations"/>
+                <filter string="Main Packages" name="main_packages" invisible="not context.get('main_package_ids')" domain="[('id', 'in', context.get('main_package_ids'))]"/>
```

Comment:
don't see this context elsewhere

Reply:
It was dropped in the mean time, thanks!